.github/workflows/velox_docker.yml

```diff
@@ -120,6 +120,12 @@ jobs:
         with:
           name: velox-arrow-jar-centos-7-${{github.sha}}
           path: /root/.m2/repository/org/apache/arrow/
+      - name: Setup tzdata
+        run: |
+          if [ "${{ matrix.os }}" = "ubuntu:22.04" ]; then
+            apt-get update
+            TZ="Etc/GMT" DEBIAN_FRONTEND=noninteractive apt-get install -y tzdata
```

Comment:
note: should add this in setup script

Reply:
https://github.com/oap-project/velox/blob/update/scripts/setup-ubuntu.sh#L93